desc/perturbations.py

```diff
@@ -251,15 +251,13 @@ def perturb(  # noqa: C901 - FIXME: break this up into simpler pieces
         scale_inv = W
         scale = np.linalg.inv(scale_inv)
 
-        f = objective.compute(x)
-
         # 1st partial derivatives wrt both state vector (x) and input parameters (c)
         if verbose > 0:
             print("Computing df")
         timer.start("df computation")
         Jx = objective.jac(x)
         Jx_reduced = Jx[:, unfixed_idx] @ Z @ scale
-        RHS1 = f + objective.jvp(tangents, x)
+        RHS1 = objective.jvp(tangents, x)
```

Comment:
what was the reason for this change again?

Reply:
Formally the `f` term shouldn't be there (its order epsilon^0 rather than epsilon^1). And also in theory, it may lead to the perturbations trying too hard to decrease force error rather than keeping force the same but moving to a new region of parameter space, leaving the solution stuck in the wrong local minima.

I re-ran some of the examples locally and it seems like removing `f` improves some cases and makes others worse. I'll revert it for now until we can be sure.